Source/Core/Core/DSP/Jit/DSPEmitter.cpp

```diff
@@ -333,7 +333,7 @@ void DSPEmitter::Compile(u16 start_addr)
   {
     m_block_links[start_addr] = m_block_link_entry;
 
-    for (u16 i = 0x0000; i < 0xffff; ++i)
+    for (size_t i = 0; i < 0xffff; ++i)
```

Comment:
The original code is from 2011 (by skidau) and used that number already (with the array being +1) - I'd guess it is a typo (or an opportunity to mess with us, if this was intentional; but I don't see an `i+1` access inside the array; so meh.)